components/EventosTimeline.vue

```diff
@@ -154,31 +166,43 @@ export default {
 .texto-calendario {
   display: block;
   float: right;
+  color: #fff;
+  margin-right: -10px;
 }
 
 .v-timeline-item {
-  padding-bottom: 0px;
   flex-direction: row-reverse !important;
 }
 
+.v-timeline-item:last-child {
+  padding-bottom: 75px;
+}
+
 .v-timeline:before {
-  background-color: #a06646 !important;
-  width: 10px;
-  right: calc(50% - 5px) !important;
+  background-color: #efb654 !important;
+  width: 9px;
+  right: calc(50% - 4.5px) !important;
   border-radius: 10px;
 }
 
 .data-dia {
-  color: #794a30;
-  font-size: 24px;
-  margin-bottom: -10px;
+  font-family: Gang of Three;
+  font-size: 30px;
+  margin-bottom: -12px;
 }
 
 .data-mes {
-  color: #a06646;
   font-size: 14px;
 }
 
+.lightOn {
+  filter: drop-shadow(0 0 0.75rem white);
```

Comment:
Talvez a luz fique muito concentrada com esse blur-radius, seria interessante aumentar um pouco.
```suggestion
  filter: drop-shadow(0 0 1rem white);
```

Reply:
Melhora mesmo!